proposals/dao/tip_119.ts

```diff
@@ -0,0 +1,116 @@
+  const gOHMHoldingPCVDeposit = contracts.gOHMHoldingPCVDeposit;
+
+  // 1. Validate gOHM ETH oracle price is valid
+  const gOhmETHPrice = (await gOhmEthOracle.read())[0];
+  // Eth price is ~$1000. gOHM price is ~$2400
+  // Therefore, gOHM price in ETH should be ~ (2400/1000) = 2.4 ETH
+  expect(toBN(gOhmETHPrice.value)).to.be.bignumber.at.least(ethers.constants.WeiPerEther.mul(1));
+  expect(toBN(gOhmETHPrice.value)).to.be.bignumber.at.most(ethers.constants.WeiPerEther.mul(5));
+
+  // 2. gOHM USD oracle price is valid
+  const gOhmUSDPrice = (await gOhmUSDOracle.read())[0];
+  expect(toBN(gOhmUSDPrice.value)).to.be.bignumber.at.least(ethers.constants.WeiPerEther.mul(1_000)); // $1000
+  expect(toBN(gOhmUSDPrice.value)).to.be.bignumber.at.most(ethers.constants.WeiPerEther.mul(5_000)); // $5000
+
+  // 3. Verify deposit and oracle added to CR
+  expect(await collateralizationOracle.isTokenInPcv(addresses.gohm)).to.be.true;
+  expect(await collateralizationOracle.depositToToken(gOHMHoldingPCVDeposit.address)).to.be.equal(addresses.gohm);
+
+  // 4. Verify pcvStats increased as expected
+  // display pcvStats
+  console.log('----------------------------------------------------');
+  console.log(' pcvStatsBefore.protocolControlledValue [M]e18 ', Number(pcvStatsBefore.protocolControlledValue) / 1e24);
+  console.log(' pcvStatsBefore.userCirculatingFei      [M]e18 ', Number(pcvStatsBefore.userCirculatingFei) / 1e24);
+  console.log(' pcvStatsBefore.protocolEquity          [M]e18 ', Number(pcvStatsBefore.protocolEquity) / 1e24);
+  const pcvStatsAfter: PcvStats = await contracts.collateralizationOracle.pcvStats();
+  console.log('----------------------------------------------------');
+  console.log(' pcvStatsAfter.protocolControlledValue  [M]e18 ', Number(pcvStatsAfter.protocolControlledValue) / 1e24);
+  console.log(' pcvStatsAfter.userCirculatingFei       [M]e18 ', Number(pcvStatsAfter.userCirculatingFei) / 1e24);
+  console.log(' pcvStatsAfter.protocolEquity           [M]e18 ', Number(pcvStatsAfter.protocolEquity) / 1e24);
+  console.log('----------------------------------------------------');
+  const pcvDiff = pcvStatsAfter.protocolControlledValue.sub(pcvStatsBefore.protocolControlledValue);
+  const cFeiDiff = pcvStatsAfter.userCirculatingFei.sub(pcvStatsBefore.userCirculatingFei);
+  const eqDiff = pcvStatsAfter.protocolEquity.sub(pcvStatsBefore.protocolEquity);
+  console.log(' PCV diff                               [M]e18 ', Number(pcvDiff) / 1e24);
+  console.log(' Circ FEI diff                          [M]e18 ', Number(cFeiDiff) / 1e24);
+  console.log(' Equity diff                            [M]e18 ', Number(eqDiff) / 1e24);
+  console.log('----------------------------------------------------');
+
+  // PCV Equity change should be neutral for this proposal
+  // expect(Number(eqDiff) / 1e18).to.be.at.least(1_000_000);
+  // expect(Number(eqDiff) / 1e18).to.be.at.most(3_000_000);
```

Comment:
would be interesting to see this bump. What's the number ?

Reply:
Should be ~$1.5M. Will uncomment now that the fix is in the other branch

Reply:
> would be interesting to see this bump. What's the number ?

PCV equity increase is $1.46M